<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grader_editor</class>
 <widget class="QWidget" name="grader_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="preview_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Previe&amp;w</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>File name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="comment_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>411</width>
     <height>89</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marks</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="marks_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QWidget" name="marks_widget" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="file_name_combo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>411</width>
     <height>27</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="gen_pdf_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Generate Pdf</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
